README.md

```diff
@@ -43,6 +43,18 @@ bin/evmone-unittests
 bin/evmone-bench
 ```
 
+### Tools
+
+#### evm-test
```

Comment:
At the bare minimum extending this section on build instruction if it is possible to only build this without evmone to be used by other projects would be useful.

Reply:
This is not possible right now. I need another round of CMake changes to re-tune build options.